### PR TITLE
butt

### PR DESCRIPTION
**Pull request by erikwoods**
Originally #13 but it wouldn't let me reopen.

### CODE REVIEW

**Comment by erikwoods** [comment]
what does the `content check` label mean?

File: _config.yml [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
 # Site settings
 title: I'll Acccept Anything
-email: your-email@domain.com
+email: progval+github@progval.net
```
Comment (progval) [inline _config.yml]: Why is my email address here?

Comment (erikwoods) [inline _config.yml]: It's just a config file. I put it there because this is your repo. I can change it to.... my email if you want. LOL.

Comment (progval) [inline _config.yml]: it's not mine

Comment (erikwoods) [inline _config.yml]: Oh. My mistake. Ummmmm..... what makes sense?

Comment (erikwoods) [inline _config.yml]: changed it.

**Comment by erikwoods** [comment]
I updated the email address yo.....

**Comment by jlu5** [comment]
Contect checks means that we're unsure about what this does / whether it meets the [guidelines](https://github.com/illacceptanything/illacceptanything#rules). Either way there's a merge conflict that needs to be fixed because I can't merge as is

**Comment by erikwoods** [comment]
@GLolol: it's been like an entire year since I made this PR. That's how merge conflicts happen. Lol. Anyways, I don't really care anymore, to be completely honest. I've lost interest in the efforts this repo was after. It's clear that progress or change takes too long. Not worth it if the maintainers aren't into it. Nahmsayin?

**Comment by jlu5** [comment]
fair enough ¯\\\_(ツ)\_/¯

**Comment by erikwoods** [comment]
I love the idea. Good luck.

**Pull request closed without merging**